tests/test_images.sh

```diff
@@ -270,6 +343,8 @@ handle_options()
 
 		build_rootfs "${distro}" "${rootfs}"
 
+		local rootfs_size=$(du -sb "${rootfs}" | awk '{print $1}')
```

Comment:
Ah, I think I get it - you are taking the `du` of the rootfs dir on the build machine itself. That's probably good enough. I was of course imagining we'd have to go look inside the image file itself to get the actual size...
Side question for @jcvenegas - we don't strip the files during the conversion from the rootfs dir to the image file itself, do we?

Reply:
@grahamwhaley  we dont that  may be a good think to do, lets open an issue for that.